README.md

```diff
@@ -1,4 +1,8 @@
 # Zookeeper Snap
+
+[![CharmHub Badge](https://snapcraft.io/charmed-zookeeper/badge.svg)](https://snapcraft.io/charmed-zookeeper)
+[![Release](https://github.com/canonical/charmed-zookeeper-snap/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-zookeeper-snap/actions/workflows/publish.yaml)
```

Comment:
We have `build & test` CI. I think that could be added here.

Reply:
I believe the problem here is that we don't have the "build & test" in the release pipeline (although maybe we should) but only in the PRs. So the workflows only include feature branches, see [here](https://github.com/canonical/charmed-zookeeper-snap/actions/workflows/ci.yaml)